src/css/application.css

```diff
@@ -604,3 +604,21 @@ body {
 .u__hidden {
   display: none;
 }
+
+.loading {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
+
+.loading__spinner {
+  background-image: url('/images/spinner.gif');
```

Comment:
How do you feel about this particular spinner? I don’t hate it, but I feel like it’s a bit small and dark—might be worth trying to spinner style we use in the `output__delayed-error-overlay` element